install/data/tyk-pump.docker.conf

```diff
@@ -0,0 +1,74 @@
+{
+  "analytics_storage_type": "redis",
+  "analytics_storage_config": {
+    "type": "redis",
+    "host": "localhost",
+    "port": 6379,
+    "hosts": null,
+    "username": "",
+    "password": "",
+    "database": 0,
+    "optimisation_max_idle": 100,
+    "optimisation_max_active": 0,
+    "enable_cluster": false
+  },
+  "purge_delay": 10,
+  "pumps": {
+    "mongo": {
+      "type": "mongo",
+      "meta": {
+        "collection_name": "tyk_analytics",
+        "mongo_url": "mongodb://localhost/tyk_analytics",
+        "collection_cap_max_size_bytes": 1048576,
+        "collection_cap_enable": true
+      }
+    },
+    "kafka": {
```

Comment:
I don't think default conf needs kafka, prom, etc. samples in the same file. Mongo should be sufficient for the default use case.

Reply:
Done.